tests/plugins/test_add_yum_repo_by_url.py

```diff
@@ -34,7 +34,7 @@ class X(object):
 def prepare():
     if MOCK:
         mock_docker()
-    tasker = DockerTasker()
+    tasker = DockerTasker(retry_times=1)
```

Comment:
A cleaner way of doing this is putting:
```
fake_docker_tasker = DockerTasker(retry_times=1)
flexmock(DockerTasker).new_instances = fake_docker_tasker
```
in `mock_docker`, as we might want to test the actual docker here with long retries